src/Catalog/Helpers/JsonSort.cs

```diff
@@ -4,8 +4,6 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
-using System.Text;
```

Comment:
I did a "remove all unnecessary references" on the solution because I was noticing a lot of them--will gladly revert this change if there's any push-back.